pretrain/dataset.py

```diff
@@ -257,8 +316,8 @@ def __init__(
         self,
         batch_size,
         sequence_length,
-        num_pages: typing.List[int],
-        tokenizer: AutoTokenizer,
+        num_pages=None,
+        tokenizer: AutoTokenizer=None,
```

Comment:
A tokenizer is required for this class to work (used on line 367), so None seems like an odd default